src/CMakeLists.txt

```diff
@@ -192,6 +196,8 @@ if(INTEL_SSE4_1)
   if (INTEL_SSE4_2)
     set(SSE4_FLAGS "${SSE4_FLAGS} -msse4.2")
   endif (INTEL_SSE4_2)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SSE4_FLAGS}")
```

Comment:
i am not sure if we want enable SSE3 and SSE4 globally. `ec_jerasure_sse{3,4}` are compiled as plugins, so user can opt out of them, but if we enable it, we force the user to use a host machine with better instruction set support than the build machine.